tests/system/feature_store/test_spark_engine.py

```diff
@@ -721,3 +729,49 @@ def test_error_is_properly_propagated(self):
                 spark_context=self.spark_service,
                 run_config=fs.RunConfig(local=False),
             )
+
+    @pytest.mark.parametrize("engine", ["spark"])
+    def test_get_offline_features_with_filter(self, engine):
+        key = "patient_id"
+        measurements = fs.FeatureSet(
+            "measurements",
+            entities=[fs.Entity(key)],
+            timestamp_key="timestamp",
+            engine="spark",
+        )
+        source = ParquetSource("myparquet", path=self.get_remote_pq_source_path())
+        fs.ingest(
+            measurements,
+            source,
+            return_df=True,
```

Comment:
Why are you passing `return_df=True` when running remotely and not trying to get the df returned from the function? Just change this to `return_df=False` (or ignore this parameter).